### DISCUSSION TITLE
Perplexity AI API

### DISCUSSION THREAD

**Discussion by Augusto271** [karakeep-app/karakeep#719] [Q&A]
Hello everyone,

So I have a hoarder unraid deployment running a local machine and I’m trying to get it to work with the perplexity ai API. The reason behind this is that since I have perplexity pro they offer me a credit per month on their API which is nice since I don’t have a powerful enough machine to run ollama reliably… I’ve managed to make the AI summary work however the tags aren’t working… any idea why and how can I fix it?

I’ve managed to make it work by replacing everything Open AI with the perplexity links and API Key.

Regards

**Comment by MohamedBassem** [comment]
Can you share the logs of the web container after adding a bookmark to understand why the tags don't work?

**Comment by Augusto271** [comment]
Hello Mohammed,

I got this in the logs should be error. I don’t understand much of programming (mostly self-taught) can you have a look?

Regards

![IMG_1327](https://github.com/user-attachments/assets/0a802cc1-8c6f-457c-a043-e5aef40359a6)
